aws-iot-device-sdk-java-samples/src/main/java/com/amazonaws/services/iot/client/sample/greengrass/BasicDiscovery.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazonaws.services.iot.client.sample.greengrass;
+
+import com.amazonaws.services.iot.client.*;
+import com.amazonaws.services.iot.client.greengrass.ConnectivityInfo;
```

Comment:
I tried your code and deserialization to `com.amazonaws.services.iot.client.greengrass.ConnectivityInfo` is failing for me, because the properties in the response are pascal case.
Thanks for this by the way.

Reply:
Is there any plans of merging this PR anytime soon? I noticed that before this comment that it has been waiting for approval since last year. I am working in a project that could benefit from it. Is there anything I could do to help?